functions/New-DbaLogin.ps1

```diff
@@ -103,6 +103,11 @@
 
         Copies logins [Login1] and [Login2] from instance sql1 to instance sql2, but enforces password and expiration policies for the new logins. New logins will also have a default database set to [tempdb] and will be created in a disabled state.
 
+    .EXAMPLE
+        PS C:\> New-DbaLogin -SqlInstance sql1 -Login domain\user
```

Comment:
You need to revert this file back to the version in the development branch. You added this update in #4163